invenio_oaiserver/alembic/5d25c1981985_add_system_created_field.py

```diff
@@ -0,0 +1,34 @@
+#
+# This file is part of Invenio.
+# Copyright (C) 2022 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Add system_created field."""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "5d25c1981985"
+down_revision = "e655021de0de"
+branch_labels = ()
+depends_on = None
+
+
+def upgrade():
```

Comment:
Can we avoid the 3 steps and the direct sql queries (which should be ok, but you bypass the compatibility layer):
```
op.add_column(
        "oaiserver_set", 
        sa.Column(
            "system_created", 
            sa.Boolean(), 
            default=True,
            server_default=sqlalchemy.sql.expression.literal(True), 
            nullable=False
        )
)
```
See [here](https://stackoverflow.com/a/16100442/675178). To be tested of course.

Reply:
Yes, it works, much better, thanks!